server/sonar-ce-task-projectanalysis/src/main/java/org/sonar/ce/task/projectanalysis/source/NewLinesRepository.java

```diff
@@ -40,6 +42,13 @@ public class NewLinesRepository {
   private final ScmInfoRepository scmInfoRepository;
   private final PeriodHolder periodHolder;
   private final Map<Component, Optional<Set<Integer>>> reportChangedLinesCache = new HashMap<>();
+  private final List<String> sfMetaLanguages = Arrays.asList("object", "object-meta.xml", "profile", "profile-meta.xml",
```

Comment:
Instead of hardcoding the metadata file suffixes, it's preferable to retrieve it from the scanner context (from property sf.sfmeta.suffixes).

Reply:
As discussed, for now, we are proceeding with hardcoded values, as related files are in different modules.